modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNodeHeader.scss

```diff
@@ -24,7 +24,6 @@ hr {
 		font-weight: 600;
 		line-height: 1.3rem;
 		overflow: hidden;
-		padding-right: 4rem;
```

Comment:
hey @igor-franca, just to confirm, this change is unrelated to the bug right?  I saw it was an necessary improvement to the nodes header, great work!

Reply:
Correct, when we are using the Arabic language. The titles of the nodes were positioned incorrectly due to this style.

I realized that it wasn't necessary even for the other languages ​​and decided to remove it.